<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePINedit</class>
 <widget class="QDialog" name="ChangePINedit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/트레이 아이콘.webp</normaloff>icon/트레이 아이콘.webp</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(150, 177, 208);</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="ChangePIN">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>26</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 63 26pt &quot;Bahnschrift SemiBold&quot;;</string>
   </property>
   <property name="text">
    <string>Change your PIN</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NewPINedit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>KoPub돋움체 Medium</family>
    </font>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmNewPINEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>KoPub돋움체 Medium</family>
    </font>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="ConfirmButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;HY헤드라인M&quot;;
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>등록</string>
   </property>
  </widget>
  <widget class="QLabel" name="NewPIN">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>13</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 63 13pt &quot;Bahnschrift SemiBold&quot;;</string>
   </property>
   <property name="text">
    <string>New PIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="ConfirmPIN">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>13</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 63 13pt &quot;Bahnschrift SemiBold&quot;;</string>
   </property>
   <property name="text">
    <string>Confirm PIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="CurrentPIN">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift SemiBold</family>
     <pointsize>13</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 63 13pt &quot;Bahnschrift SemiBold&quot;;</string>
   </property>
   <property name="text">
    <string>Current PIN</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CurrentPINedit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>KoPub돋움체 Medium</family>
    </font>
   </property>
   <property name="maxLength">
    <number>6</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="CancleButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 9pt &quot;HY헤드라인M&quot;;
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
